css/styles.css

```diff
@@ -659,3 +666,479 @@ p#modal-close {
 .modal-btn {
   margin-top: 2rem;
 }
+
+/* Desktop Version */
+@media screen and (min-width: 768px) {
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    max-width: 100rem;
+    margin: 0 auto;
+    margin-top: 1rem;
+    top: 35%;
+    left: 167px;
+    padding: 2.5rem;
+    position: absolute;
+  }
+
+  .header {
+    background: var(--white);
+    position: absolute;
+    font-family: "Poppins", sans-serif;
+    font-size: 19px;
+    font-weight: 700;
+    text-align: center;
+    line-height: 20px;
+    order: 0;
+    flex: -17px;
+    flex: none;
+    height: 61px;
+    display: flex;
+    width: 80%;
+    top: 7px;
+    left: 118px;
+    z-index: 1;
+  }
+
+  .logo {
+    text-decoration: none;
+    color: var(--color-accent);
+    font-size: 1.8rem;
+    font-weight: 700;
+  }
+
+  .headline-section {
+    height: 100vh;
+    background: var(--white) url(../images/background.svg);
+    object-fit: fill;
+    background-size: 100%;
+    background-position: 8px;
+    background-position-y: 62px;
+    top: -1px;
+    margin-bottom: -20px;
+    border-bottom-left-radius: -66px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: 10rem;
+  }
+
+  .headline-text {
+    padding-left: -0.8rem;
+    font-size: 1rem;
+    max-height: 100px;
+    line-height: 22px;
+    margin: -1px -27px;
+    margin-left: -3px;
+    max-width: 47rem;
+  }
+
+  .nav-menu li a {
+    color: #000;
+    list-style-type: none;
+    font-weight: 420;
+    font-size: 1.1rem;
+  }
```

Comment:
- [ ] Good job implementing the  change request by the previous code reviewer , but when you apply the changes you introduced the misalignment
![Capture](https://user-images.githubusercontent.com/80458199/229279218-a2ccce2e-0b46-40ae-b40d-09a1eb9347d7.PNG)
 of the nav links toward the logo, Kindly make sure the nav link are well aligned 👍